seshdash/views.py

```diff
@@ -690,4 +699,48 @@ def historical_data(request):
         context_dict['active_site'] = active_site
         context_dict['sort_keys'] = sort_data_dict.keys()
         context_dict['sort_dict'] = sort_data_dict
+        print sort_data_dict
         return render(request, 'seshdash/historical-data.html', context_dict);
+    
+@login_required
+def get_measurements_values(request):
+    if request.method == 'POST':
+        results = {}
+        dropdown1_choice_results =[]
+        dropdown2_choice_results =[]
+        choice_drop_1 = request.POST.get('choice1','')
+        choice_drop_2 = request.POST.get('choice2','')
+        active_id = request.POST.get('active_site_id','')
+        active_site = Sesh_Site.objects.filter(id=active_id)
+        active_site_name = active_site [0]
+        current_site = active_site_name.site_name
+        SI_unit = Daily_Data_Point.UNITS_DICTIONARY
+        SI_unit1 = SI_unit[choice_drop_1]
+        SI_unit2 = SI_unit[choice_drop_2]
+        client = Influx('test_db')
+        values_drop_1 = client.get_measurement_bucket(choice_drop_1,'10m','site_name',current_site,'1d')
+        values_drop_2 = client.get_measurement_bucket(choice_drop_2,'10m','site_name',current_site,'1d')
+        dropdown1_values = values_drop_1[0]
+        dropdown2_values = values_drop_2[0]
+        for dropdown1_values in values_drop_1:
+            dropdown1_choice_results.append(dropdown1_values['mean'])
+
+        for dropdown2_values in values_drop_2:
+            dropdown2_choice_results.append(dropdown2_values['mean'])
+        results['drop1'] = dropdown1_choice_results
+        results['drop2'] = dropdown2_choice_results
+        results['SI_unit1'] = SI_unit1
+        results['SI_unit2'] = SI_unit2
+        return HttpResponse(json.dumps(results))
+    else:
+        return HttpResponseForbidden()
```

Comment:
I don't think forbidden is the correct response. This needs to be more  like an empty response.